src/Elevator.java

```diff
@@ -15,9 +15,22 @@ public void run() {
 			RequestData requestData = scheduler.processRequest();
 			
 			int diff = requestData.getCurrentFloor() - requestData.getRequestedFloor();

```

Comment:
I think `diff` might be unnecessary since RequestData has an attribute called `move` which would indicate if its `UP` or `DOWN`